File: 算法基础课(修订版)/循环/P14073/data/.upload_cpp/user_acm.cc
#include <iostream>
#include <vector>
#include <string>
using namespace std;

class Solution {
public:
    string solve(vector<int>& leftWeights, vector<int>& rightWeights) {
        // 请在这里实现
        return "";
    }
};

int main() {
    int n, m;
    cin >> n >> m;

    vector<int> leftWeights(n);
    vector<int> rightWeights(m);

    for (int i = 0; i < n; ++i) {
        cin >> leftWeights[i];
    }
    for (int i = 0; i < m; ++i) {
        cin >> rightWeights[i];
    }

    Solution solution;
    cout << solution.solve(leftWeights, rightWeights) << endl;

    return 0;
}

File: 算法基础课(修订版)/循环/P14073/data/.upload_python/user_acm.py
class Solution:
    def solve(self, left_weights, right_weights):
        # 请在这里实现
        return ""


if __name__ == "__main__":
    n, m = map(int, input().split())
    left_weights = list(map(int, input().split()))
    right_weights = list(map(int, input().split()))

    solution = Solution()
    result = solution.solve(left_weights, right_weights)

    print(result)
